modules/core/src/main/scala/org/tessellation/cli/incremental.scala

```diff
@@ -0,0 +1,23 @@
+package org.tessellation.cli
+
+import org.tessellation.ext.decline.decline._
+import org.tessellation.schema.SnapshotOrdinal
+import org.tessellation.sdk.config.AppEnvironment
+import org.tessellation.sdk.config.AppEnvironment.{Mainnet, Testnet}
+
+import com.monovore.decline.Opts
+import com.monovore.decline.refined._
+import eu.timepit.refined.auto._
+import eu.timepit.refined.types.numeric.NonNegLong
+
+object incremental {
+  val lastFullGlobalSnapshot: Map[AppEnvironment, SnapshotOrdinal] = Map(
+    Mainnet -> SnapshotOrdinal(0L), // TODO: set before mainnet release
+    Testnet -> SnapshotOrdinal(736766L)
+  )
+
+  val lastFullGlobalSnapshotOpts: Opts[Option[SnapshotOrdinal]] = Opts
+    .argument[NonNegLong]("lastFullGlobalSnapshot")
+    .map(SnapshotOrdinal(_))
+    .orNone
```

Comment:
`orNone` + `getOrElse` is technically `.withDefault(SnapshotOrdinal(0L)` but I see that defaulting doesn't make sense for `AppEnvironment.Dev` so let's leave it as it is. Just wanted to point out `withDefault` method for future use-cases.